elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/internal/guarantee/GuaranteeService.java

```diff
@@ -105,21 +109,90 @@ public boolean isRegisterCompleteSuccess(final Collection<Integer> shardingItems
         }
         return true;
     }
-    
+
     /**
      * Judge whether job's sharding items are all completed.
      *
      * @return job's sharding items are all completed or not
      */
     public boolean isAllCompleted() {
         return jobNodeStorage.isJobNodeExisted(GuaranteeNode.COMPLETED_ROOT)
-                && configService.load(false).getShardingTotalCount() <= jobNodeStorage.getJobNodeChildrenKeys(GuaranteeNode.COMPLETED_ROOT).size();
+                && configService.load(false).getShardingTotalCount()
+                <= jobNodeStorage.getJobNodeChildrenKeys(GuaranteeNode.COMPLETED_ROOT).size();
     }
-    
+
```

Comment:
Could you revert the unrelated changes in this file?

Reply:
Whitespace can not revert, no diff, git can not push, could be problem of line break format, my format is LF. Do I need to handle it in other ways?

Reply:
Got it. Let me handle it later.